content/applications/services/fsm/default_warehouse.rst

```diff
@@ -26,17 +27,23 @@ than one warehouse in your database.
 User account
 ------------
 
-To set up a default warehouse for a specific user, go to :menuselection:`Settings --> Users -->
-Manage users`, open a user, then go to the :guilabel:`Preferences` tab, scroll down to
-:guilabel:`Inventory`, and select the default warehouse from the drop-down menu.
+To set up a default warehouse for a specific user, open the **Settings** application, then go to
+:menuselection:`Users --> Manage users`, select a user, then go to the :guilabel:`Preferences` tab.
```

Comment:
I think the current content is better
`go to :menuselection:`Settings --> Users -->`
(I know you initially left it like that :) )

Reply:
I reverted it back to how it was, to me both options are fine. With the "go to (:menuselection:Settings --> Users -->`)" version, it wasn't super clear during the 1st review which "settings" we were talking about. That's why we added "Settings application" version. Lmk if it changes anything or you :)